targets/csharp/source/Plugins/CloudScript/source/PlayFabFunctionContexts.cs

```diff
@@ -38,32 +39,29 @@ public class FunctionContext<TFunctionArgument>
         /// </summary>
         /// <param name="request">The request incoming to an Azure Function from the PlayFab server</param>
         /// <returns>A new populated <c>FunctionContext</c> instance</returns>
-        public static async Task<FunctionContext<TFunctionArgument>> Create(HttpRequestMessage request)
+        public static async Task<FunctionContext<TFunctionArgument>> Create(HttpRequest request)
```

Comment:
At a high level, I feel like you should be using the ITransport plugin, which performs HTTPS interactions for you in an implementation agnostic way.
You access it through PluginManager, and it allows the customer to replace the ITransport plugin with their own implementation.

Reply:
I think that it's especially relevant since you seem to be replacing it here, which is exactly what ITransport is meant to avoid.
ALSO, ITransport will vary from one platform to another, and the default that comes with the SDK is guaranteed to work.